### PR TITLE
Add support for hardware in the new RoDI

### PR DESCRIPTION
**Pull request by garyservin**
Added support for
- Pixel RGB led
- Light Sensor
- Line following sensors (not new hardware)

Also added more tests and docstrings for methods =)

### CODE REVIEW

File: rodi.py [hunk 1]
```diff
@@ -88,21 +150,119 @@ def see(self):
...
+        Returns the luminosity of the ambient with values from 0 to 1023
+        '''
+        url = self._build_url(
+            self.LIGHT_METHOD,
+            []
+        )
+        response = requests.get(url)
+        return json.loads(response.content)
+
+    def led(self, state):
```
Comment (humitos) [inline rodi.py:192]: what are the possible values for state here?

**Comment by humitos** [comment]
It looks amazing. I don't have a new RoDI hardward, so there is no way to test if from my side.

Did you test this results in a real context?

After your confirmation I will accept this PR.

Thanks a lot for this collaboration.

File: rodi.py (hunk 1 above)
Comment (garyservin) [inline rodi.py:192]: Added information

**Comment by garyservin** [comment]
Yes, I've tested this with the new version and it works! I also used this (hence the added main stuff) to do QA for the new ones I'm building!

BTW If you come to Py you may get one of the new ones =)

**Pull request merged**